src/Microsoft.DotNet.CodeAnalysis/Analyzers/PinvokeAnalyzer.cs

```diff
@@ -0,0 +1,157 @@
+using System;
```

Comment:
They are missing the copyright header. 

You could run the code formatter tool on this